src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  padding: 10px 0;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.link-logo {
+  margin-left: 50px;
+  height: 40px;
+  text-decoration: none;
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  margin: 0 50px 0 0;
+}
+
+a {
```

Comment:
Don't use tag names for styling. It's better to use classes to apply styles to elements. For example, instead of styling all `a` tags, consider using a class like `.nav__link` and apply your styles to this class.